setup.py

```diff
@@ -4,6 +4,11 @@
 
 from setuptools import setup, find_packages
 
+# If incremental is not present then setuptools just silently uses v0.0.0 so
+# let's import it and fail instead.
+import incremental
+
+
 setup(
```

Comment:
might be worth doing a `pipx run setup-py-upgrade .` too

Reply:
Welp, I was trying to do a quick i-lost-my-momentum-but-still-have-stuff-to-do-but-i-will-release-now-anyways release but...  while we're reworking the project and defining the release process and automating publishing...  `:]`